package.json

```diff
@@ -114,6 +114,7 @@
   },
   "dependencies": {
     "@serverless/enterprise-plugin": "^1.2.0",
+    "@serverless/cli": "^0.1.0",
```

Comment:
It'll be good to maintain an alphabetical order (otherwise later it'll conflict with automated version bumps)